cmd/tk/main.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/grafana/tanka/pkg/tanka"
 )
 
-var interactive = term.IsTerminal(int(os.Stdout.Fd()))
+var interactive = term.IsTerminal(int(os.Stderr.Fd()))
```

Comment:
zerolog outputs to stderr, this is the relevant fd to check

Reply:
This is used elsewhere though (to tell whether or not the pager should be activated), does checking stdout and stderr do the same thing, or should it be two different vars?

Reply:
that's a good point, I guess the error one should check stderr and the other one can keep on looking at stdout